docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,220 @@
+
+A.2/ If the editorial rules of the targeted journal permit it, the scheme used by the Astropy project
+(e.g. `Astropy V2.0 <https://arxiv.org/pdf/1801.02634.pdf>`_) should be used in priority:
+
+- 'The Gammapy Collaboration', and the list of primary paper contributors in order of contribution agreed per consensus with the development team and the Coordination Committee, as '(Primary Paper Contributors)'
+- the members of the Coordination Committee, as '(Gammapy Coordination Committee)'
+- the list of contributors of each release since the last Gammapy publication, ordered by alphabetical order, as '(Gammapy Contributors)'
+
+
+A comment on the author list composition should be added. Extracted from the Astropy project, one can precise:
+
+"The author list has three parts: the authors that made significant contributions to the writing of
+the paper in order of contribution, the members of the Gammapy Project Coordination Committee,
+and contributors to the Gammapy Project in alphabetical order. The position in the author list does not
+correspond to contributions to the Gammapy Project as a whole. A more complete list of contributors
+to the core package can be found in the package repository and at the Gammapy team webpage."
+
+C/ The corresponding author is the 'Gammapy Coordination Committee', associated with its usual mailing list.
+
+D/ Acknowledgement
+
+One has the freedom to precisely mention acknowledgements associated with the publication. 
+In practice, it is recommended to precise the grants or fellowships given to some authors.
+One should in any case always acknowledge the Astropy project, to which we are affiliated.
+
+Contributions in conferences
+-----------------------------
+
+The section concerns any contribution in conferences (a talk, a poster and their associated proceedings) related to
+the Gammapy project itself. It does not concern a technical or scientific work that uses Gammapy as an open library.
+In this case, the standard citation scheme should be used.
+
+As the length of the author list is generally a constrain, the author list can be reduced to the shortlist of authors,
+followed by 'The Gammapy Collaboration' associated with a link to the Gammapy team webpage.
+
+If there is a corresponding author, the 'Gammapy Coordination Committee', associated with its usual mailing list, is
+used. Concerning the acknowledgement, the Astropy project should always be mentioned.
+
+
+4. Storage in metadata files
+==================================
+
+Depending on the software repository, different metadata files are used in the eco-system of Open Source research
+software.
+
+The file ``codemeta.json`` is used by HAL and Software Heritage, and recommended by
+`ESCAPE <https://escape2020.pages.in2p3.fr/wp3/eossr/v0.3.2/metadata.html>`_ or
+`EOSC <https://www.eosc-pillar.eu/news/illustrating-benefits-eosc-research-communities-germany>`_.
+
+The file ``citation.cff`` is used by Zenodo and GitHub.
```

Comment:
Not necessary!!! Have a look on tributor library. From @vuillaut :
![mermaid-diagram-20220523183724](https://user-images.githubusercontent.com/16781593/171587832-86abb6a0-e64e-41e6-abb3-9e95c4474ded.png)

Reply:
* I'm not an author of tributors
* the schema is simply trying to clarify the current metadata landscape and converters
* `CITATION.cff` may be the reference file, as you can see on the schema it can be the source of conversion to other formats, however, it's a less complete schema than others